core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -41,8 +41,12 @@
         <picture>
           <source type="image/webp" ng-srcset="<[$ctrl.getStaticImageUrl('/logo/288x128_logo_white.webp')]>">
           <source type="image/png" ng-srcset="<[$ctrl.getStaticImageUrl('/logo/288x128_logo_white.png')]>">
-          <img ng-src="<[$ctrl.getStaticImageUrl('/logo/288x128_logo_white.png')]>" class="oppia-logo" ng-class="'oppia-logo-wide'">
+          <img ng-src="<[$ctrl.getStaticImageUrl('/logo/288x128_logo_white.png')]>"
+               class="oppia-logo"
+               ng-class="'oppia-logo-wide'"
+               alt="">
```

Comment:
@jimbyo  Seems like I've reviewed this PR earlier! is this an expected change for this PR?

Reply:
@DubeySandeep Oh yes sorry. This PR is a branch from https://github.com/oppia/oppia/pull/9613 and once that PR merges, I'll fix the merge conflicts in this PR.

Reply:
I found [this comment ](https://github.com/oppia/oppia/pull/9692#issuecomment-651271559) which explains why I see a similar code here. @jimbyo I would suggest you add these kinds of general comments related to PR which is useful for reviewers in the PR header (before the checklist).

Reply:
Go it